src/RecoveryServices/RecoveryServices.SiteRecovery/Properties/Resources.resx

```diff
@@ -480,6 +480,9 @@ Recommended Action: To cleanup the resources created by Test failover run the Te
   <data name="NicNotFoundInVM" xml:space="preserve">
     <value>Nic "{0}" not found in VM.</value>
   </data>
+  <data name="IPConfigNotFoundInVMNic" xml:space="preserve">
+    <value>IP Config "{0}" not found in VM NIC.</value>
```

Comment:
why is this added in the middle. Add at the end please

Reply:
updated and moved to end